.github/workflows/vcpkg.yml

```diff
@@ -122,6 +135,8 @@ jobs:
           done
 
       - name: Save cache dependencies
+        # Don't save if it's the exact same
+        if: steps.cache-restore.outputs.cache-matched-key != format('{0}-{1}', matrix.os, hashFiles('vcpkg-info/*'))
```

Comment:
It already not save if it have the same cache id. I prefer the step will calculate it, and not by this line.